data/language.json

```diff
@@ -23,7 +23,7 @@
     "name": "Ukrainian",
     "contributors": [
       "STANDART SCP",
-      "Lambda Project"
+      "n3k0chka"
```

Comment:
Did you change your nickname yourself? Or is it someone else's name?

Reply:
its mine, check previous pull requests